app/controllers/idv/review_controller.rb

```diff
@@ -9,7 +9,7 @@ class ReviewController < ApplicationController
 
     def confirm_idv_steps_complete
       return redirect_to(idv_session_url) unless idv_profile_complete?
-      return redirect_to(idv_address_url) unless idv_address_complete?
+      return redirect_to(idv_phone_url) unless idv_address_complete?
```

Comment:
Does it still make sense to use `idv_address_complete?` here? That looks like something we could tear out of `Idv::Session` now.

Reply:
`idv_address_complete?` is defined in that controller:
```
def idv_address_complete?
  idv_session.address_mechanism_chosen?
 end
```

I left it and didn't rename it because:
- we're still kinda sorta verifying address here
- the address mechanism is still important, it gets set when phone/usps verification is actually done and I didn't want to remove it.
- didn't want the PR to start ballooning, but if we should move or remove it, I can do that in a separate PR.

Reply:
@jmhooper Is this OK to leave for now?

Reply:
:+1:. Let's just keep in mind that address method selection looks like a good place to start an Idv::Session refactor if somehow we ever get bored.